src/constants.ts

```diff
@@ -31,6 +31,8 @@ export class Messages {
   public static itemDeleted = Messages.baseSingular + 'deleted';
   public static itemsReturned = Messages.basePlural + 'returned';
   public static itemsRented = Messages.basePlural + 'rented';
+  public static itemAlreadyRented = 'Item already rented';
```

Comment:
Can we change these strings to be more friendly?
"Item is already rented!"
"Item is available"
These may be better options, but you know better than I.